index.rst

```diff
@@ -1,13 +1,15 @@
 Welcome to the UBports documentation!
 =====================================
 
-Welcome to the official documentation of the UBports project!
+Welcome to the official documentation of the UBports project! We are a team of engineers, designers, community managers, translators, students and volunteers who together bring to life Ubuntu Touch, a mobile operating system focused on ease of use, privacy, and convergence. We are also lovers of Free software and open collaboration. You can find us all over the web, but Github provides the best starting point for planning your involvement. If you are new, head to get.ubuntu.com to get started
 
-UBports develops the mobile phone operating system Ubuntu Touch. Ubuntu Touch is a mobile operating system focused on ease of use, privacy, and convergence.
+Ubuntu Touch is a mobile operating system focused on ease of use, privacy, and convergence. It features a tablet-oriented user interface with access to a desktop interface when a keyboard or mouse is connected.
 
 On this website you find guides to :doc:`install Ubuntu Touch on your mobile phone <userguide/install>`, :doc:`use Ubuntu Touch <userguide/dailyuse/index>`, :doc:`develop Ubuntu Touch apps <appdev/index>`, :doc:`port Ubuntu Touch to an Android handset <porting/introduction/index>` and :doc:`learn more about system components <systemdev/index>`. If this is your first time here, please consider reading our :doc:`introduction <about/introduction>`.
 
-If you want to help improving this documentation, :doc:`the Documentation contribute page <contribute/documentation>` will get you started.
+If you want to help improving this documentation you can, in many different ways including: Fork this repository.  Make progress from where I've stalled with updates/corrections/additions from others from the Issues Todo list.  Submit a document translation in a different language. :doc:`the Documentation contribute page <contribute/documentation>` will get you started.
+
+If you would like to get feedback before you commit, you may join UBports Doc Group @ubports_docs on telegram.
```

Comment:
again, if it's needed it should go on the docs contrib page not start page. aaaaand, (this may be just a "me problem", but) I struggle with TG groups that become big and wide focus, with a lot of unrelated chitchat, so, invite-only worked better for me personally ... if however there is some consensus in the writers group for open-invite, then, well, who am I to stop it, but then keep in mind that we decided to deprecate the *docs* group when we merged with the *writers* group, so that would be the one to link ... if we want any public link ... let's maybe clarify that question in the group first

Reply:
I like people to have a place to start a discussion; from those talks good coming out, individuals working alone trying to figure out puzzles won't help anyone. So yes, TG maybe not be the best solution, but I like the Red Hat approach of 'Open Organization Culture.' (Read about that)

Example: Among all the pages on the Ubports website, the UI/UX is shine with a "UI group", people end there with "what can I help".

Five joined "QA Testing" last week because we told them to join and help. 2 repeated the exact copy lionel wrote to ask, "who needs a test". There were super hot leads!

TG is not that bad when it comes to search. I use the search like an engine, it's hards, but the community builds upon this; changing the communication channel will be a hard move.

Another Example: One SUPER HOT technical writer knew I was looking for help last summer, and he was ready to write anything. He contacted me because Dalton said on QA I was looking for support on language.

But I didn't even know what I needed back than, and I was wanted to focus on different things; as a group, he might find other people to partner with!

Reply:
Mozila or Redhat are far dream, but what about KDE?

I wanted to build something like this: https://community.kde.org/Get_Involved

But Dalton asked me to focus on Docs. Not the best imho, but ok.

Reply:
But the KDE link goes to some doc too, doesn't it? :thinking: 

We have our doc and we also have a contributing section. This file, [index.rst](https://docs.ubports.com/en/latest/index.html) is the aquivalent of [https://community.kde.org/Main_Page](https://community.kde.org/Main_Page). Your link is the aquivalent of [https://docs.ubports.com/en/latest/contribute/index.html](https://docs.ubports.com/en/latest/contribute/index.html).

And the first starting page - in my opinion - should provide a short overview, a guide on how to find information. It should not contain the actual information itself.

This means a link "help writing for this doc" is fine. The full instructions how to do that is not. That needs to go to the contributions part.

Reply:
Out problem we mixing with 'People Docs' and 'Software Doc', while everyone else not. From nextcloud to every open source.

Reply:
Can you please explain that? I am not sure I understand what you try to say.

Reply:
Think about the all this a huge funnel of people, breaking hard content should happen by steps, from People to Tech.
When new "employee" get to new job he don't get the production line specification, he get where he eat launch and other people birthday (kidding) but I hope you get the idea. These are two spread layers of communication

Reply:
I agree with that should be part of the contribution part, but this page is very big landing page, which everyone start in, bounce rate are low then any page. But that is where we are stuck. Writing. So I was wanted to put it up their first.